### PR TITLE
Add new package azure-iot-hub

### PR DESCRIPTION
**Pull request by fratster**
Add hooks in yaml and powershell files to include azure-iot-hub in release pipeline

### CODE REVIEW

**Comment by cartertinney** [comment]
Don't we need to add a `.bumpversion.cfg` as well?

File: scripts/build-release.ps1 [hunk 1]
```diff
@@ -35,6 +35,11 @@ function Build {
         Version = $env:nspkg_version_part
     }
 
+    $packages["azure-iot-hub"] = [PSCustomObject]@{
+        File = "setup.py"
```
Comment (cartertinney) [inline scripts/build-release.ps1]: >"setup.py" [](start = 15, length = 10)

This should be `"azure\iot\hub\constant.py"`

**Comment by cartertinney** [comment]
And a setup.cfg for that matter?

File: scripts/build-release.ps1 (hunk 1 above)
Comment (fratster) [reply scripts/build-release.ps1]: got it thx

File: azure-iot-device/.bumpverion.cfg [hunk 2]
```diff
@@ -1,7 +1,5 @@
 [bumpversion]
-current_version = 2.0.0-preview.11
-parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)-preview\.(?P<preview>\d+)
-serialize = {major}.{minor}.{patch}-preview.{preview}
-
-[bumpversion:part:preview]
+current_version = 2.0.0
```
[the remark this answers is not in the document]
Comment (fratster) [reply azure-iot-device/.bumpverion.cfg]: sorry was testing, I reverted it

**Pull request merged**